src/styleUtils.js

```diff
@@ -64,19 +63,18 @@ define(
 		);
 	};
 	
-	// Uses the Flash based downloadify library to save files to the local file system
+	// Uses the FileSaver.js library to save files to the local file system
 	var saveCsl = function (
 			cslCodeOrData,
 			styleId,
 			comment // optional, if comment is included cslCodeOrData
 			        // must be a CSLEDIT_Data instance
 			) {
 		var dialog = $('<div title="Save CSL Style">' +
```

Comment:
The indendenting makes this very hard to read. Can you fix that, also it wasn't changed by you? Actually, I would suggest to write this in one line and make the comment below, e.g.
```js
 	var saveCsl = function (cslCodeOrData, styleId, comment) {
 	// comment is optional,
 	// if comment is included cslCodeOrData must be a CSLEDIT_Data instance
 	 	var dialog = $('<div title="Save CSL Style">' +
```
Maybe even the following lines with variable declaration could be easier to read if we add some `var` and put them separate them more.